db/xml/items/items.xml

```diff
@@ -0,0 +1,31 @@
+<!-- 
+    Detective-style problem-solving magical rpg
+	Copyright (C) 2023 Rainy Day Studio
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program.  If not, see <https://www.gnu.org/licenses/>. 
+-->
+<db>
+    <item id="test" name="ShinyPebble">
+        <ingredient name="Shiny">
+			<amount>2</amount>
+			<volume>5</volume>
+		</ingredient>
+    </item>
+	<item id="test" name="NewtEye">
+        <ingredient name="Newt">
+			<amount>3</amount>
+			<volume>3</volume>
```

Comment:
Try to stay consistent with what you make an attribute and what you make an xml node. Imo amount and volume can be attributes (makes it easier to query) like what you did in the requirements.